test/BUILD.gn

```diff
@@ -106,6 +106,7 @@ test("brave_unit_tests") {
     "//brave/chromium_src/chrome/browser/signin/account_consistency_disabled_unittest.cc",
     "//brave/chromium_src/components/autofill/core/browser/autofill_experiments_unittest.cc",
     "//brave/chromium_src/components/history/core/browser/sync/brave_typed_url_sync_bridge_unittest.cc",
+    "//brave/chromium_src/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc",
```

Comment:
I see `"//brave/chromium_src/components/history/core/browser/sync/brave_typed_url_sync_bridge_unittest.cc",`
which already includes `src/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc`.

Currently `//components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc` is compiled into upstream tests which looks wrong. I think the right approach is to:
1. nullify `//components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc` by putting an empty file at `//brave/chromium_src` so it won't be compiled in upstream tests (add copyright and a comment explaining the reasoning).
2. keep using it directly via `src/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc` from our `brave_typed_url_sync_bridge_unittest.cc`.

Reply:
Thanks, this is fixed at https://github.com/brave/brave-core/pull/16705/commits/e6fc2aed6d0ee96f5dac5e1e036feb1a01a63f0b